app/frontend/Settings/Global/LanguageDropDown.svelte

```diff
@@ -1,4 +1,5 @@
 <select bind:value={language} on:change>
+  <option value="">{$t`System`}</option>
```

Comment:
merged

Label change not yet done

Reply:
I couldn't work out how to do substitutions in labels (I have all sorts of problems with that in the front end recurrence code). I think i used that label because it was already translated?

Reply:
```
let languageName = ...
$t(`System - ${languageName}`)
```
should work.

The translation library will use the English string, until there are translations. We also have scripts to extract this string and mark it for future translation.

Reply:
@NeilRashbrook FYI: this doesn't work for me. If I set "System", then at the next start, the dropdown selected value is empty (the list is still there), and it uses English instead of German. Whereas by default (no preference) set, it uses German on my system, as expected.

Reply:
That would be the erroneous `|| null` in `localStorage.setItem("ui.locale", language || null);` on line 100 of `l10n.ts`. Sorry about that.